CONTRIBUTING.md

```diff
@@ -4,24 +4,63 @@ If you would like to contribute to this project by modifying/adding to the progr
+3. Change into your new project folder.
+ * From the command line: `cd ProjectPorcupine`
+4. [optional]  Add the upstream repository to your list of remotes.
+ * From the command line: `git remote add upstream https://github.com/TeamPorcupine/ProjectPorcupine.git`
+5. Create a branch for your new feature.
+ * From the command line: `git checkout -b my-feature-branch-name`
+6. [optional]  It is recommended that you rebase your branch on top of the latest master, to minimize excess commit messages.
+ * From the command line:  
+     `git fetch upstream master`  
+     `get rebase upstream/master`  
+6. Make your changes.
  * Avoid making changes to more files than necessary for your feature (i.e. refrain from combining your "real" pull request with incidental bug fixes). This will simplify the merging process and make your changes clearer.
  * Very much avoid making changes to the Unity-specific files, like the scene and the project settings unless absolutely necessary. Changes here are very likely to cause difficult to merge conflicts. Work in code as much as possible. (We will be trying to change the UI to be more code-driven in the future.) Making changes to prefabs should generally be safe -- but create a copy of the main scene and work there instead (then delete your copy of the scene before committing).
-5. Commit your changes and push your branch to your fork.
-  * You may want to pull in the lastest project games from the upstream
-    master before pushing your own changes, to ensure that you are up
-    to date.
-6. Make a "Pull Request" from your branch here on Github.
+7. Commit your changes and push your branch to your fork.
+  * From the command line:  
+    `git add Assets/my-changed-file.cs`  
+    `git add Assets/my-other-changed-file.cs`  
+    `git commit -m "A descriptive commit message"`  
+    `git push origin my-feature-branch-name`  
+8. Make a "Pull Request" from your branch here on Github.
   * Include screenshots demonstrating your change if applicable.
-7. For a video tutorial, please see: https://www.youtube.com/watch?v=-N4Cghw0l2Q
+9. For a video tutorial, please see: https://www.youtube.com/watch?v=R2fl17eEpwI
+
+# Resolving Merge Conflicts
+
+Depending on the order that Pull Requests get processed, your PR may result in a conflict and become un-mergable.  To correct this, do the following from the command line:  
+  
+Switch to your branch: `git checkout my-feature-branch-name`  
+Pull in the lastest upstream changes: `git pull upstream master`  
+Find out what files have a conflict: `git status`  
+
+Edit the conflicting file(s) and look for a block that looks like this:  
+    `<<<<<<< HEAD`  
+    `my awesome change`  
+    `=======`  
+    `some other person's less awesome change`  
+    `>>>>>>> some-branch`  
+
+Replace all five (or more) lines with the correct version (yours, theirs, or
+a combination of the two).  ONLY the correct content should remain (none of 
+that "<<<<< HEAD" stuff.)
```

Comment:
Maybe add something here showing the potential outcome, like:

Example after merge:
```
    `some other person's less awesome change`
    `my awesome change`
```